okd/src/use_okd_assets.sh

```diff
@@ -43,6 +43,14 @@ replace_assets(){
     # update the infra pods for crio
     sed -i 's,pause_image .*,pause_image = '"\"${pod_image}\""',' "packaging/crio.conf.d/10-microshift_${UNAME_TO_GOARCH_MAP[${arch}]}.conf"
 
+    # kube proxy is required for flannel
+    kube_proxy_okd_image_with_hash=$(oc adm release info --image-for="kube-proxy" "${okd_url}:${okd_releaseTag}")
+    echo "kube-proxy ${kube_proxy_okd_image}"
+    kube_proxy_okd_image_name="${kube_proxy_okd_image_with_hash%%@*}"
+    echo "kube-proxy image name ${kube_proxy_okd_image_name}"
+    kube_proxy_okd_image_hash="${kube_proxy_okd_image_with_hash##*@}"
+    echo "kube-proxy image hash ${kube_proxy_okd_image_hash}"
```

Comment:
What are we trying to do with the name manipulation? Maybe expanding on it in the comment would be helpful.
Can we remove the `echo`'s?

Reply:
Yes I will put comment why it is required.